JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -155,6 +155,10 @@
         public static dump(object: any): string {
             var objectTypeDump: string = Object.prototype.toString.call(object);
             var propertyValueDump: string = JSON.stringify(object);
+            if (objectTypeDump === "[object Error]") {
+                propertyValueDump = "{ stack: '" + object.stack + "', message: '" + object.message + "', name: '" + object.name + "'";
```

Comment:
Nvm. I see that you are adding the `stack` below. Maybe add that as a note before line 159?